### PR TITLE
Add sun moon pokémon

### PR DESCRIPTION
**Pull request by geniusgordon**

### CODE REVIEW

File: data/ru.json [hunk 1]
```diff
@@ -719,5 +719,86 @@
...
+	"Дартрикс",
+	"Десиджуай",
+	"Литтен",
+	"Торракат",
+	" Инсинероар",
+	"Попплио",
+	"Бриан",
+	"Примарина",
+	"Пикипек",
+	null,
```
Comment (SamVerschueren) [inline data/ru.json]: Quite a lot of translations are missing here.

Comment (geniusgordon) [inline data/ru.json]: I got the names from [Bulbapedia](http://bulbapedia.bulbagarden.net/wiki/Main_Page)
Didn't notice that some of the Russian names are missing 🤔

Comment (geniusgordon) [inline data/ru.json]: I'll try to look up somewhere else

**Comment by sindresorhus** [comment]
Ping @geniusgordon

**Comment by geniusgordon** [comment]
Sorry, I really can't find those missing Russian names 😭

**Comment by sindresorhus** [comment]
Maybe @zverbeta can help.

**Comment by zverbeta** [comment]
@geniusgordon Hi, when i did pull request it was complete pokemon names. All of these names were taken from bulbapedia site

**Comment by geniusgordon** [comment]
I looked up the names from  bulbapedia too.
But some of the russian names are missing, like the Ultra beasts

**Comment by sindresorhus** [comment]
@geniusgordon Can you try some other sites? There must be some site that has them all.

**Pull request closed without merging**